### ISSUE TITLE
merge of saving

### ISSUE THREAD

**Issue by dalex1212** [ruiramos/greatdj#10]
hi,

first of all great tool all other tools which i tested were buggy or lacked of features.

following feature request to have all i need :)
I have created playlist and save and share it.
then my buddy open it and add song b and saves.
I still have the version open in a tab when i sent it and add song c and save.
when i refresh the playist only song c is added and the change of my budyy is lost.
would be great before a save that last version is load and then only the delta is saved.

thx
Alex

**Comment by ruiramos** [comment]
Hi Alex,

Thanks for the feedback.
You're right, I should totally implement some kind of diff system to update the playlists based on the deltas.
If you're looking for realtime collaboration though, have you tried party mode? If both you and your friend enable it, you'll see each other changes as they happen. (I understand this still doesn't solve the problem).

Cheers, 
Rui

**Comment by dalex1212** [comment]
Yeah party mode is great e.g. We use it when you share a office and what to
play music and each one can stop,  skip and add... :)

So when will the diff system happen :)
Am 27.08.2015 18:19 schrieb "Rui Ramos" <notifications@github.com>:
